src/clr-angular/popover/dropdown/dropdown-trigger.spec.ts

```diff
@@ -49,5 +49,23 @@ export default function(): void {
     it('declares itself to the DropdownFocusHandler', function(this: Context) {
       expect(this.getClarityProvider(DropdownFocusHandler).trigger).toBe(this.clarityElement);
     });
+
```

Comment:
It will be good to test that the component will get its default aria-label value, not only that is `undefined`. 

This is also true `'' !== undefined`